.github/workflows/python.yml

```diff
@@ -0,0 +1,44 @@
+name: Python linters
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - pontoon/**.py
+      - .github/workflows/python.yml
+  pull_request:
+    branches:
+      - master
+    paths:
+      - pontoon/**.py
+      - .github/workflows/python.yml
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Dependencies
+        run: |
+          pip install -U pip
+          pip install -r requirements/lint.txt
+      - name: flake8
+        run: flake8 pontoon
+
+  black:
+    runs-on: ubuntu-latest
```

Comment:
Is there a benefit in splitting the two jobs? One job with multiple steps would reuse the checkout and the running image.

Reply:
The benefit is that it runs both if one fails. That's a thing I really despise about the travis setup we have right now.

Also, it's easier to understand which tool failed if one doesn't have to dig that up in the logs.

Reply:
Makes sense. I assume it comes with a cost, i.e. more time to run the whole test?

Reply:
I followed up on this conversation in the bug, seems worthwhile to document the rationales there.